### PR TITLE
chore: 🤖 toggle translation based on state

### PR DESCRIPTION
**Pull request by DhariniJeeva**
this pr toggles the translation message based on states

## Screen
Untitled 2
shots (if appropriate):
Untitled 3

## How to Test
1. use ENT binary, go to session recording
2. the recording state and completed state should show correct messages after reapply

:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:

- [x] I have added before and after screenshots for UI changes
- ~[ ] I have added JSON response output for API changes~
- [x] I have added steps to reproduce and test for bug fixes in the description
- ~[ ] I have commented on my code, particularly in hard-to-understand areas~
- [x] My changes generate no new warnings
- ~[ ] I have added tests that prove my fix is effective or that my feature works~

### CODE REVIEW

File: ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js [hunk 1]
```diff
@@ -48,4 +51,20 @@ export default class ScopesScopeSessionRecordingsSessionRecordingChannelsByConne
...
   }
+
+  /**
+   * Reapplies storage policy dates to session recording
+   * @param {SessionRecordingModel}
+   */
+  @action
+  @notifyError(({ message }) => message, { catch: true })
+  @notifySuccess(({ state }) =>
+    state === STATE_SESSION_RECORDING_AVAILABLE
```
Comment (ZedLi) [inline ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js:62]: If the session is in an error or unknown state, should it also say later?

**Review by ZedLi — APPROVED** [review]

File: ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js:62]: thanks for raising this- checking with BE, for now, we show the manage dropdown, not sure if the user can still reapply at that state

**Review by dvonanderson — APPROVED** [review]

File: ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/app/routes/scopes/scope/session-recordings/session-recording/channels-by-connection.js:62]: update: yes, we will show it as later

**Review by lisbet-alvarez — APPROVED** [review]
LGTM

**Pull request merged**